tests/components/plex/test_browse_media.py

```diff
@@ -77,10 +78,61 @@ async def test_browse_media(hass, hass_ws_client):
     result = msg["result"]
     assert result[ATTR_MEDIA_CONTENT_TYPE] == "server"
     assert result[ATTR_MEDIA_CONTENT_ID] == DEFAULT_DATA[CONF_SERVER_IDENTIFIER]
-    assert len(result["children"]) == len(mock_plex_server.library.sections())
+    assert len(result["children"]) == len(mock_plex_server.library.sections()) + len(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)